src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/help/Get-AzureRmRecoveryServicesBackupJob.md

```diff
@@ -71,13 +71,13 @@ This script polls the first job that is currently in progress until the job has
 
 ### -BackupManagementType
 Specifies the Backup management type.
-Currently, only AzureVM is supported.
+Currently, only AzureVM, AzureStorage is supported.
```

Comment:
@siddharth7 nit: the accepted values down below include a few more values than the two mentioned here -- is this expected?

Reply:
@cormacpayne Yes, the ones that are mentioned below denote all the backupManagement types https://github.com/Azure/azure-powershell/blob/2501703f1271bba24ea6f8358bd4f44cd00955d3/src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Models/CommonModels/Enums.cs#L46  but we only provide support to AzureVM and AzureStorage in this context